tool/tsh/tsh.go

```diff
@@ -366,6 +368,7 @@ const (
 	userEnvVar             = "TELEPORT_USER"
 	addKeysToAgentEnvVar   = "TELEPORT_ADD_KEYS_TO_AGENT"
 	useLocalSSHAgentEnvVar = "TELEPORT_USE_LOCAL_SSH_AGENT"
+	globalTshConfigEnvVar  = "TELEPORT_GLOBAL_TSH_CONFIG"
```

Comment:
This is a backport of a merged PR, so I don't think you need to make a change, but I would have made this `globalTSHConfigEnvVar` to be consistent with our other vars and Go conventions.

Reply:
Ah, I'll keep it in mind in the future, perhaps there will be a good opportunity to revisit the naming.